native/packages/react-native-bpk-component-spinner/stories.js

```diff
@@ -76,19 +76,15 @@ const getThemedButtons = () => (
 );
 
 storiesOf('BpkSpinner', module)
-  .add('docs:primary', () => (
-    <BpkSpinner />
-  ))
-  .add('docs:light', () => (
-    <View style={styles.dark}>
-      <BpkSpinner type="light" />
+  .add('docs:default', () => (
+    <View>
+      {generateAllSpinnerTypes()}
     </View>
```

Comment:
Decided this didn't need to be separated out or the docs page would have loads of screenshots. TextInput has them all in one so I'm mirroring that here.